### PR TITLE
chore: update workflows for security center v2

### PR DESCRIPTION
**Pull request by subfuzion**
## Description

Updates workflows for an upcoming series of related pull requests.

Context: The problem I need to solve is to isolate the v2 failures from v1 (I don't want changes made in v2 to run v1 tests and vice versa, but package.json dep updates should trigger both). It's not clear to me if it's necessary to have two separate workflows or if the path stuff can somehow be manipulated in a single file or if breaking these out is the cleaner approach.

### CODE REVIEW

File: .github/workflows/security-center-snippets-v2.yaml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: security-center-snippets-v2
+on:
+  push:
+    branches:
+    - main
+    paths:
+    - 'security-center/snippets/v1/**'
```
Comment (grayside) [inline .github/workflows/security-center-snippets-v2.yaml]: suggestion: Should this be targeting v2?

**Review by grayside — COMMENTED** [review]
Initial feedback, pending further discussion to get to a review resolution.

File: .github/workflows/security-center-snippets.yaml [hunk 2]
```diff
@@ -56,7 +56,7 @@ jobs:
         access_token_lifetime: 600s
     - uses: actions/setup-node@v4.0.0
       with:
-        node-version: 16
+        node-version: 18
```
[the remark this answers is not in the document]
Comment (subfuzion) [reply .github/workflows/security-center-snippets.yaml:59]: Ah, thanks for that -- let me update both to v20 then.

File: .github/workflows/security-center-snippets-v2.yaml [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - labeled
+    paths:
+      - 'security-center/snippets/v2/**'
+      - 'security-center/snippets/package.json'
+      - '.github/workflows/security-center-snippets.yaml'
```
[the remark this answers is not in the document]
Comment (subfuzion) [reply .github/workflows/security-center-snippets-v2.yaml:33]: I caught that and force-pushed already; beat you to it lol

File: .github/workflows/security-center-snippets-v2.yaml [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+        env:
+          GCLOUD_ORGANIZATION: 1081635000895
+      - name: upload test results for FlakyBot workflow
+        if: github.event.action == 'schedule' && always()
+        uses: actions/upload-artifact@v3
+        env:
+          MOCHA_REPORTER_OUTPUT: "${{github.run_id}}_sponge_log.xml"
+        with:
+          name: test-results
+          path: security-center/snippets/${{ env.MOCHA_REPORTER_OUTPUT }}
```
[the remark this answers is not in the document]
Comment (subfuzion) [reply .github/workflows/security-center-snippets-v2.yaml:93]: Good suggestion, thank you!

File: .github/workflows/security-center-snippets-v2.yaml (hunk 1 above)
Comment (subfuzion) [reply .github/workflows/security-center-snippets-v2.yaml]: I caught that and force-pushed already; beat you to it :)

File: .github/workflows/security-center-snippets-v2.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (subfuzion) [reply .github/workflows/security-center-snippets-v2.yaml]: Good catch, thanks!

**Comment by subfuzion** [comment]
@grayside Converting to draft while I address the ESM errors introduced by renovate. See b/3322624.

**Comment by subfuzion** [comment]
@grayside Ready now. PTAL.

**Comment by subfuzion** [comment]
@grayside Change of plan. I was continuing work from the original contributor, but have concluded that this is really unnecessary. We don't actually distinguish between v1 and v2 when we run system tests, so no point in two workflows running that run all the tests for now (we can always add this back later if that's desired). However, since I used this PR to fix a bug that does exist thanks to a renovate update, I'll submit a new PR just for that. Thanks for the review!

**Comment by subfuzion** [comment]
See #3791

**Pull request closed without merging**